src/plugins/gimbal/gimbal_manager.cpp

```diff
@@ -0,0 +1,69 @@
+#include "plugins/gimbal/gimbal_manager.h"
+#include "gimbal_manager_impl.h"
+
+namespace mavsdk {
+
+GimbalManager::GimbalManager(System& system) : PluginBase(), _impl{new GimbalManagerImpl(system)} {}
+
+GimbalManager::~GimbalManager() {}
+
+GimbalManager::Result GimbalManager::set_attitude(float tilt_rate, float pan_rate,
+        float tilt_deg, float pan_deg, int operation_flags,
+        uint8_t id)
+{
+    return _impl->set_attitude(tilt_rate, pan_rate, tilt_deg, pan_deg, operation_flags, id);
+}
+
+void GimbalManager::set_attitude_async(float tilt_rate, float pan_rate,
+        float tilt_deg, float pan_deg, int operation_flags,
+        uint8_t id, result_callback_t callback)
+{
+    _impl->set_attitude_async(tilt_rate, pan_rate, tilt_deg, pan_deg, operation_flags, id, callback);
+}
+
+GimbalManager::Result GimbalManager::track_point(float x, float y, uint8_t id)
```

Comment:
Let's hold off with this functionality. I would rather add features one by one as we can test them. We have no experience with tracking yet (and it would require a camera interacting with the gimbal).

Reply:
I'll just comment it out for now